src/utils/githubApi.ts

```diff
@@ -107,6 +107,13 @@ export class GithubRemote {
   }
 }
 
+/**
+ * Returns a GithubRemote object to the sentry release registry.
+ */
+export function getRegistryGithubRemote(): GithubRemote {
+  return new GithubRemote('getsentry', 'sentry-release-registry');
+}
```

Comment:
nit: I think I get why you put it here, but since it won't be used by any other github api consumers other than in the context of registry interactions, I'd vote for moving it to `src/utils/registry.ts`